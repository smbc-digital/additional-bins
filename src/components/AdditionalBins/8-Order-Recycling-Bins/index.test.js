import React from 'react'
import Enzyme, { mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { OrderRecyclingBins } from './index'
//import renderer from 'react-test-renderer'
import getPageRoute from '../../../helpers/pagehelper'


Enzyme.configure({ adapter: new Adapter() })
describe('OrderRecyclingBins',() => { 

    it('should push to correct page', () => {
        // Arrange
        window.location.assign = jest.fn()
		const history = { push: jest.fn() }
        const context = {
            doYouRecycle: {
                value: 'test',
                isValid : true
            }
        }

		// Act
        const wrapper = mount(<OrderRecyclingBins history={history} context={context} />)
    
        // Assert
        wrapper.find('form').simulate('submit')
        expect(window.location.assign).toBeCalledWith('https://www.stockport.gov.uk/start/additional-recycling-bins')
    })
})