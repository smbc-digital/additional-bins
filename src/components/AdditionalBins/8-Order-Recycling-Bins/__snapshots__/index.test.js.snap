// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderRecyclingBins snapshot renders correctly 1`] = `
<form>
  <div
    className="ineligible-warning"
  >
    <div
      className="header-container"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-exclamation"
        />
      </div>
      <h1>
        It looks like you need additional recycling bins
      </h1>
    </div>
    <h2>
      What you can do next
    </h2>
    <p>
      You can order additional recycling bins so that you
    </p>
    <ul>
      <li>
        recycle as much as possible
      </li>
      <li>
        have more space in your black bin for your non-recyclable waste
      </li>
    </ul>
    <p>
      Use the button below to order additional recycling bins.
    </p>
    <a
      className="button-primary"
      href="https://www.stockport.gov.uk/start/additional-recycling-bins"
    >
      Order additional recycling bins
    </a>
  </div>
</form>
`;
